test/tender_transaction_test.py

```diff
@@ -0,0 +1,12 @@
+import shopify
+from test.test_helper import TestCase
+
+class TenderTransactionTest(TestCase):
```

Comment:
can we add a test for the `#show` action as well? Generally I prefer for the tests to cover all of the available controller actions so they act as documentation for the package

Reply:
Talked about this on slack, there's currently only an `index` route on the controller. In the future, we could add scoping on orders if a use-case appears.